tools/autograd/derivatives.yaml

```diff
@@ -1396,6 +1443,7 @@
 
 - name: zero_(Tensor(a!) self) -> Tensor(a!)
   self: zeros_like(grad)
+  result: auto_linear
```

Comment:
Could this be auto_element_wise?

Reply:
No because element-wise does not handle the inplace case.

Reply:
We prefer this one to be inplace. So linear is prefered here.